detect_secrets/main.py

```diff
@@ -28,7 +28,7 @@ def main(argv=None):
     args = parse_args(argv, parserBuilder)
 
     if args.verbose:  # pragma: no cover
-        log.set_debug_level(args.verbose)
+        log.set_debug_level(3)
```

Comment:
`3` is the value that corresponds to the DEBUG logging level of the `mapping` dictionary within `_set_debug_level`.